readthedocs/templates/projects/project_import.html

```diff
@@ -209,6 +209,17 @@ <h1>{% trans "Import a Repository" %}</h1>
         </div>
       {% endblock %}
 
+      <h3>{% trans "Filter by Own Repositories" %}</h3>
+      <li
+        class="remote-org"
+        data-bind="click: function () { $root.set_filter_own(); }">
+        <!-- TODO: show all the connected accounts here with corresponding avatar -->
+        <span>humitos (GitHub)</span>
```

Comment:
Here, I want to list all of the social account connected and list them here.

I was thinking on creating a new JS object for this (like `Organization`) and do a query at init to populate the list. Sounds like a good idea?

Reply:
I solved this by using Django allauth templates to generate each link.